scripts/d.rast.edit/d.rast.edit.py

```diff
@@ -79,6 +79,11 @@
 import atexit
 import grass.script as grass
 
+from grass.script.setup import set_gui_path
+set_gui_path()
+
+from gui_core.wrap import ClientDC, Menu, Panel
+
```

Comment:
Currently, the compilation fails, this probably needs to be in the try except below.

Reply:
Fixed it in the [3819af9](https://github.com/OSGeo/grass/pull/1085/commits/3819af92e73871ca5fbd197d6ede0e4a1a117ca3).